static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -150,6 +150,11 @@ const userIsInCorrectCohort = (
     }
 };
 
+const isValidCohort = (cohort: string): boolean =>
+    ['OnlyExistingSupporters', 'OnlyNonSupporters', 'Everyone'].includes(
```

Comment:
with Flow, you can use `$Keys` when you want to use an enum type at both compile-time and run-time: https://flow.org/en/docs/types/utilities/#toc-keys

Reply:
ahh, nice!